packages/dotcom-ui-shell/src/components/DocumentHead.tsx

```diff
@@ -46,12 +46,17 @@ const DocumentHead = (props: TDocumentHeadProps) => (
     {/* packaging */}
     <link
       rel="icon"
+      type="image/svg+xml"
+      href="https://www.ft.com/__origami/service/image/v2/images/raw/ftlogo-v1%3Abrand-ft-logo-square-coloured?source=update-logos"
```

Comment:
Can we reuse the `imageServiceIconURL` function that is used in the other examples here?

Reply:
Sure thing :-)